application.py

```diff
@@ -1,33 +1,36 @@
 from flask import Flask, request, jsonify, render_template, url_for, redirect
-import requests
-import os
-import requests
 import json
 from dotenv import load_dotenv
-import xml.etree.ElementTree as ET
+import boto3
+
 
 #flask command does this on its own. we need to do it explicity for gunicorn
 load_dotenv('.env')
 
-with open("config.json", "r") as read_file:
-    data = json.load(read_file)
+#get config.json from s3 bucket
+s3 = boto3.resource('s3')
+obj = s3.Object('ezproxy-lookup-stage','config.json')
+body = obj.get()['Body'].read()
+data = json.loads(body)
 
 app = Flask(__name__)
 
 @app.route("/", methods=['POST'])
 def index_post():
         # check for form data
-        if 'url' in request.form:
+        url = request.form.get('url')
+        if url:
+            print(request.form['url'])

```

Comment:
I suspect you are doing some debug type logging here. I'd suggest using a logging library for that instead of `print()` if you want this info logged in production.